Header header

# Orientation as Quaternion (in degrees)
geometry_msgs/Quaternion orientation

# Angular Velocity (in degrees per second)
geometry_msgs/Vector3 angular_velocity

# Linear Acceleration (in metre per second squared)
geometry_msgs/Vector3 linear_acceleration

# Raw magnetometer data (in micro-Tesla)
geometry_msgs/Vector3 raw_magnetometer

# Calibration Status
uint8 sys
uint8 gyro
uint8 accel
uint8 mag